### PR TITLE
React Capstone: Metrics Webapp

### PR DESCRIPTION
**Pull request by VSeray**
This app consumes the [Financial Modeling Prep API](https://site.financialmodelingprep.com/developer/docs).

 Home:

- When the page loads, the app shows the list of market categories including forex, cryptocurrencies, stocks, and ETFs.

- When the user clicks (or taps) on a category, the application navigates to the details page showing a list of all current prices.

Details Page:

- The app retrieves data from the API to show a list of market prices.

- When the user clicks on the "Back" button (<), the user navigates to the home page.

**[Video Preview](https://www.loom.com/share/e2e186f66b4146118ab7a7a3995fd836)**

**[Live link](https://vseray-metrics-webapp.netlify.app/)**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/VSeray/Metrics-Webapp/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (Meltrust) [inline README.md:57]: - [ ]  In your "Acknowledgments" section, kindly include credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

That way, we will meet this particular requirement 💪

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka+One&family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
```
Comment (Meltrust) [inline src/App.css]: - [ ] OPTIONAL: I recommend you implement a rule that changes the cursor to a pointer when a button is hovered (like the back button).  That way the UX will vastly improve!

File: src/components/Home.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { categories } from './data';
+import {
+  getForex, getCryptos, getStocks, getEFTs,
+} from '../redux/actions/markets';
+import '../styles/Home.css';
+
+const Home = () => {
```
Comment (Meltrust) [inline src/components/Home.js]: According to the project requirements:

image


- [ ] Please, kindly implement a filter for your main categories (Forex, Majors, Cryptos, Stocks, ETF iShares... etc).  

## Explanation

This filter should be some parameter, like a number (e.g. capitalization), letters (e.g. search box for the categories names), or a drop-down list.  The main goal is that only the categories that apply appear when filtered.


## Hints

You could create a component for the filtered rendering, and add it to your main page.  Then, connect it to the Redux store to manage the state of the filtering.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @VSeray,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- The App works well✔️ 
- Nice use of English on your video ✔️ 
- Nice use of React Router ✔️ 
- Nice Redux implementation and code organization! ✔️ 
- Beautiful styling ✔️ 

You can do it

## Required Changes ♻️

- [ ] On your video, please kindly make sure it contains a highlight about a piece of code that you find interesting or you are proud of making. 💪 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by er-contreras — APPROVED** [review]
Hi @VSeray, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

-Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**